### PR TITLE
Improve packaging for debian/ubuntu.

### PR DESCRIPTION
**Pull request by hosiet**
This commit would refine .deb packaging instructions for debian and ubuntu.

Testing on Debian Stretch. Should work for Ubuntu 16.04 and newer systems.

### CODE REVIEW

File: debian/watch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+version=4
+opts="filenamemangle=s%(?:.*?)?v?(\d[\d.]*)\.tar\.gz%jwilm-$1.tar.gz%" \
+ https://github.com/hluk/CopyQ/tags \
+ (?:.*?/)?v?(\d[\d.]*)\.tar\.gz debian uupdate
```
Comment (FadeMind) [inline debian/watch]:
```
uupdate
```
@hosiet  a typo at 4 line?

Comment (hosiet) [reply debian/watch]: Nope, 100% correct.

Comment (hosiet) [reply debian/watch]: Oops I saw a little mistake, hold on a sec. --> done.

Comment (FadeMind) [reply debian/watch]: s

**Comment by hluk** [comment]
Nice, thanks!

**Pull request merged**